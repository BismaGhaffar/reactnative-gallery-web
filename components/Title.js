import styled from 'styled-components'

const Title = styled.h1`
  text-align: center;
  color: #fff;
  @media (min-width: 481px) {
    font-size: 35px;
  }
  @media (min-width: 769px) {
    font-size: 45px;
    display: ${props => (props.hidexs ? 'none' : 'block')};
  }
`

export default Title
