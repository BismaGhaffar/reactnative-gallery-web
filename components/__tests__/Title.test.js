import React from 'react'
import renderer from 'react-test-renderer'
import Title from '../Title'

it('Title can be created', () => {
  const comp = renderer.create(<Title />)
  expect(comp).toBeDefined()
})

it('<Title /> toMatchSnapshot', () => {
  const tree = renderer.create(<Title />).toJSON()
  expect(tree).toMatchSnapshot()
})
