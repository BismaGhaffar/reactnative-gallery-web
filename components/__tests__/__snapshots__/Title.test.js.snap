// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> toMatchSnapshot 1`] = `
<h1
  className="Title-s9pqsj-0 jqUncq"
/>
`;
